cns/restserver/util.go

```diff
@@ -787,6 +783,18 @@ func (service *HTTPRestService) validateIPConfigsRequest(
 	return podInfo, types.Success, ""
 }
 
+// validateDefaultIPConfigsRequest validates the request for default IP configs request
+func (service *HTTPRestService) validateDefaultIPConfigsRequest(_ context.Context, ipConfigsRequest *cns.IPConfigsRequest) (respCode types.ResponseCode, message string) {
```

Comment:
should we rename to validateInfraIPConfigsRequest to match contract?

Reply:
fwiw this validator will run on all ipconfigs requests, so I think that leaving it as "default" should suffice.